pom.xml

```diff
@@ -3,26 +3,110 @@
+
+                <version.spring>5.0.5.RELEASE</version.spring>
+                <version.spring.core>1.2.0.RELEASE</version.spring.core>
+                <version.spring.boot>2.0.2.RELEASE</version.spring.boot>
+
+                <version.camunda>7.10.0</version.camunda>
+                <version.camunda.bpm.spring.boot>3.2.0</version.camunda.bpm.spring.boot>
+                <version.camunda.bpm.extensions>1.2</version.camunda.bpm.extensions>
+                <version.camunda.spin>1.6.7</version.camunda.spin>
+                <version.resteasy.spring.boot>3.0.0.Final</version.resteasy.spring.boot>
+                <version.resteasy.jaxrs>3.0.12.Final</version.resteasy.jaxrs>
+                <version.javax.servlet.api>3.1.0</version.javax.servlet.api>
+                <version.javax.ws.rs.api>2.1.1</version.javax.ws.rs.api>
+                <version.org.webjars.bootstrap>2.3.2</version.org.webjars.bootstrap>
+                <version.org.jboss.spec>1.0.0.Final</version.org.jboss.spec>
+                <version.wildfly>11.0.0.Final</version.wildfly>
+                <version.wildfly.maven.plugin>2.0.0.Final</version.wildfly.maven.plugin>
+                <version.jboss-as-maven-plugin>7.9.Final</version.jboss-as-maven-plugin>
+
+                <version.junit>4.12</version.junit>
+                <version.hamcrest.json>0.2</version.hamcrest.json>
+
         </properties>
 
+        <developers>
+                <developer>
+                        <name>Holger Hagen</name>
+                </developer>
+                <developer>
+                        <name>Konstantin Klaeger</name>
+                </developer>
+                <developer>
+                        <name>Bernd Breier</name>
+                </developer>
+                <developer>
+                        <name>Joerg Heffner</name>
+                </developer>
+                <developer>
+                        <name>Benjamin Fuernrohr</name>
+                </developer>
+        </developers>
+
+        <scm>
+                <url>http://github.com/taskana/taskanaadapter/tree/master</url>
+                <connection>scm:git:git://github.com/taskana/taskanaadpter.git</connection>
+                <developerConnection>scm:git:ssh://github.com:taskana/taskanaadapter.git</developerConnection>
+        </scm>
+
+        <licenses>
+                <license>
+                        <name>The Apache License, Version 2.0</name>
+                        <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+                </license>
+        </licenses>
+        <dependencyManagement>
+                <dependencies>
+                        <dependency>
+                                <groupId>org.springframework.boot</groupId>
+                                <artifactId>spring-boot-dependencies</artifactId>
+                                <version>${version.spring.boot}</version>
+                                <type>pom</type>
```

Comment:
I would prefer to add these (and the javadoc plugin) in a separate task, because inclusion of these plugins creates tons on errors and almost all source files must be modified. These action would pollute this PR, therefore

Reply:
Will be done in the context of TSK-973